vuepress/docs/v6.3.2/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -42,100 +46,102 @@ customUi: <h2>Hi from Example Widget</h2>
 ```
 
-5. Now that you've published your bundle to git you can create the Kubernetes custom resource for it.
+5. Now that you've published your bundle to Git you can create the Kubernetes custom resource for it.
 
-Install the bundler if you haven't previously done so.
+Install the bundler if you haven't previously done so
 ``` sh
 npm install -g  @entando/entando-bundler@6.3.2
 ```
 
-Next generate the custom resource for your bundle. Run the `entando-bundler from-git` command and provide your remote git repository URL via the `--repository` option and the correct namespace via `--namespace`. You can also provide a thumbnail for your bundle with `--thumbnail-file` or `--thumbnail-url`.
+To generate the custom resource for your bundle run the `entando-bundler from-git` command, then provide your remote Git repository URL via the `--repository` option and the correct namespace via `--namespace`. You can also provide a thumbnail for your bundle with `--thumbnail-file` or `--thumbnail-url`.
 
 ``` sh
-entando-bundler from-git --name=example-bundle --namespace=entandp --repository=https://your/remote/repository.git --dry-run > example-bundle.yaml
+entando-bundler from-git --name=example-bundle --namespace=entando --repository=https://your/remote/repository.git --dry-run > example-bundle.yaml
 ```
 
-Next you can apply this definition to Kubernetes. You may need to first transfer the file to your VM, e.g using `multipass transfer`.
+Now you can apply this definition to Kubernetes. You may need to first transfer the file to your VM (e.g using `multipass transfer`).
 
 ```
 kubectl -n entando apply -f example-bundle.yaml
 ```
 
-You can confirm the presence of your custom resource via `kubectl get EntandoDeBundle -n entando`
+You can confirm the presence of your custom resource with the command `kubectl get EntandoDeBundle -n entando`.
 
 ## Install the bundle into an application
-Now you can go to the `App Builder` → `Component Repository` and install your bundle. You should see your bundle in the list and when you click `Install` you can select your preferred version if the bundle contains more than one.
+Your bundle should appear in `App Builder` → `Component Repository` in your Entando instance. Clicking `Install` should allow version selection if your bundle has multiple iterations.
 
-At this point the Entando platform will download and install the components contained in the bundle. Once complete you should see the `Install` button change to give you an option to `Uninstall` that specific version. You can also navigate to `Components` → `Micro Frontends & Widgets` and find your custom widget under the `User` section.
+The Entando platform will then download and install the components contained in the bundle. Once complete you should see the `Install` button change to give you the option to `Uninstall` that specific version. If you navigate to `Components` → `Micro Frontends & Widgets` you should find your custom widget within the `User` section.
```

Comment:
Once complete, you should see the `Install` button change to give you the option to `Uninstall` that specific version. If you navigate to `Components` → `Micro Frontends & Widgets`, you should find your custom widget within the `User` section.
-couple of commas added